Makefile.main

```diff
@@ -115,6 +98,25 @@ docker-push:
 		echo "Successfully uploaded image."; \
 	done; \
 
+# this rule must be run only after tagged image was created by `docker-push`
```

Comment:
Then fail if the current image wasn't tagged.

Reply:
it will fail during `docker tag`. With error like:
```
Error response from daemon: No such image: bla:v1
Can't tag image.
```
then script stops and doesn't try to upload.

Reply:
I think this new rule should depend on the original `:docker-push`; that way, the dependency of the previous one will be programmatically ensured

Reply:
good idea! Thanks! Fixed.